<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventEditDialog</class>
 <widget class="QDialog" name="EventEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe 黑体 Std R</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/white.png</normaloff>:/icon/white.png</iconset>
  </property>
  <widget class="QLabel" name="eventNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string>Event name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="addEventLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add event</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>325</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2099</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1970</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="startTimeEdit">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>325</y>
     <width>85</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="displayFormat">
    <string>HH:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDateEdit">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>385</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2099</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1970</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>2</number>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endTimeEdit">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>385</y>
     <width>85</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>333</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="toLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="eventNameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string>accessible</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>565</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>725</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>  Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/cancel2.png</normaloff>:/icon/cancel2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="eventBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>65</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/event_grey.png</normaloff>:/icon/event_grey.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="countdownBtn">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>65</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/star_grey.png</normaloff>:/icon/star_grey.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="birthdayBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>65</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/birthday-cake_grey.png</normaloff>:/icon/birthday-cake_grey.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="anniversaryBtn">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>65</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/badge_grey.png</normaloff>:/icon/badge_grey.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="allDayLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>275</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All day</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>725</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>  Save</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="splitLine" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>500</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="DetectAreaAllDay" name="allDayDetectArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>265</y>
     <width>500</width>
     <height>45</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="allDayCheckBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="descriptionTextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>605</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="DetectAreaFrom" name="fromDetectArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="DetectAreaTo" name="toDetectArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="eventLabel">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>140</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
  </widget>
  <widget class="QLabel" name="countdownLabel">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Countdown</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthdayLabel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Birthday</string>
   </property>
  </widget>
  <widget class="QLabel" name="anniversaryLabel">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Anniversary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/delete.png</normaloff>:/icon/delete.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <zorder>fromDetectArea</zorder>
  <zorder>toDetectArea</zorder>
  <zorder>allDayDetectArea</zorder>
  <zorder>eventNameLabel</zorder>
  <zorder>addEventLabel</zorder>
  <zorder>startDateEdit</zorder>
  <zorder>startTimeEdit</zorder>
  <zorder>endDateEdit</zorder>
  <zorder>endTimeEdit</zorder>
  <zorder>fromLabel</zorder>
  <zorder>toLabel</zorder>
  <zorder>eventNameLineEdit</zorder>
  <zorder>locationLineEdit</zorder>
  <zorder>locationLabel</zorder>
  <zorder>descriptionLabel</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>eventBtn</zorder>
  <zorder>countdownBtn</zorder>
  <zorder>birthdayBtn</zorder>
  <zorder>anniversaryBtn</zorder>
  <zorder>allDayLabel</zorder>
  <zorder>fakeButton</zorder>
  <zorder>splitLine</zorder>
  <zorder>descriptionTextEdit</zorder>
  <zorder>eventLabel</zorder>
  <zorder>countdownLabel</zorder>
  <zorder>birthdayLabel</zorder>
  <zorder>anniversaryLabel</zorder>
  <zorder>fakeButton2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DetectAreaAllDay</class>
   <extends>QWidget</extends>
   <header location="global">detectareaallday.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DetectAreaTo</class>
   <extends>QWidget</extends>
   <header location="global">detectareato.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DetectAreaFrom</class>
   <extends>QWidget</extends>
   <header location="global">detectareafrom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
